File: CS/DataService/DataServiceDbContext.cs
using Microsoft.EntityFrameworkCore;

namespace DataService
{
    public class DataServiceDbContext : DbContext
    {
        public DataServiceDbContext(DbContextOptions<DataServiceDbContext> options) : base(options)
        {
        }

        public DbSet<OrderItem> OrderItems { get; set; }
    }
}

File: VB/DataService/DataServiceDbContext.vb
Imports Microsoft.EntityFrameworkCore

Namespace DataService

    Public Class DataServiceDbContext
        Inherits DbContext

        Public Sub New(ByVal options As DbContextOptions(Of DataServiceDbContext))
            MyBase.New(options)
        End Sub

        Public Property OrderItems As DbSet(Of OrderItem)
    End Class
End Namespace
